### PR TITLE
Extended README with contribution information

### PR DESCRIPTION
**Pull request by javagl**
This addresses https://github.com/KhronosGroup/glTF-Project-Explorer/issues/14

Preview at https://github.com/javagl/glTF-Project-Explorer#contributions 

Comments or suggestions?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,50 @@ Tool to provide a filterable registry of glTF community projects.
...
+as described below.
+ 
+The JSON properties that can be contained in the project information 
+are summarized in the following table:
+
+| Property | Type | Description |
+|------|--------|-------------|
+| `name` | `string` | The name of the project. This will be the title of the project card, and turned into a link, leading to the URL that is given as the `link` property |
+| `link` | `string` (URL) | The URL of the homepage of the project.  |
+ | `description` | `string` | A short description of the project. This string can contain basic [Markdown](https://en.wikipedia.org/wiki/Markdown) formatting. |
```
Comment (weegeekps) [inline README.md]: The parser we're using supports GitHub flavored Markdown, so the [documentation Github provides](https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax) may be more useful than the Wikipedia page.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,50 @@ Tool to provide a filterable registry of glTF community projects.
...
+| `license` | `string[]` | An array of strings describing the licenses under which the project is published. The strings should be the [license identifiers according to the SPDX license list](https://spdx.org/licenses/).<br> <br> Examples of licenses are `Apache-2.0`, `BSD-3-Clause`, or `MIT`. |
+| `language` | `string[]` | An array of strings indicating the programming languages of the project, where applicable. <br> <br> Examples of languages are `JavaScript`, `C++`, `Java`, or `TypeScript` |
+| `inputs`<br>`outputs` | `string[]`<br>`string[]`  | Arrays of strings listing the supported file types. These fields should at least contain the string `glTF 1.0` or `glTF 2.0`, indicating which glTF version is supported.<br> <br> Further file types may be listed for viewers or converters, to indicate the supported input- and output types. This is usually done by listing the file extension - for example, `OBJ`, `FBX`, or `COLLADA`. For universal viewers or converters, the string `Multiple` can be used instead of listing all file extensions. 
+
+All properties except for the `name` are optional. 
+
+An example of the JSON description of a project is shown here:
+ ```
```
Comment (weegeekps) [inline README.md]: It might be good to add `json` to the end of this line to enable the syntax highlighting for readability.

**Comment by javagl** [comment]
I have added syntax highlighting - it shows properly in Atom, but not on Github... (?)

I also updated markdown link. By the way, I noticed that there are some errors in the data file related to that: In the `input`/`output` fields, some projects did have a markdown link saying "Multiple", linking to a site where the formats have been listed. This link was broken during the conversion (and I'll go though that ASAP). 

But it raises one more general question: Which fields should allow markdown? I think restricting it to **only** allow it in the description makes sense. (If someone wants to add a link the the supported `input`/`output` formats, for example, it can be added there).

**Comment by weegeekps** [comment]
Right now we only parse the markdown within the description, and I agree that restricting it to there makes sense. I think it would also make sense to make a validator script that we can run on PRs using GitHub Actions to require all PRs containing data file modifications to be valid. I'm going to add an issue for this; I don't think we'll get to this soon but it seems useful.

**Comment by weegeekps** [comment]
Also, I don't know how we want to handle merges? I figure approver handles the merge? Let me know.

**Comment by javagl** [comment]
Assuming that the latter refers to the "external" PRs with updates of the data file: I think that generally, there shouldn't be much to "review", so approve+merge could be done quickly. 

The point about some sort of validity check also came to my mind. On the one hand, I'm not sure whether setting up the infrastructure for that is worth the effort: There may be a few updates when the explorer gains some visibility, and many people update their project information. But after that, I think there may not be much more than 1-2 new projects per week...? On the other hand: Considering that the JSON data file is "live", a broken file (due to some JSON parsing error caused by a missed `,` or `"`) would be critical and might immediately make the page inaccessible...

**Pull request merged**